internal/controller/contract.go

```diff
@@ -407,12 +414,8 @@ func (c *ContractController) replaceGatewayEventInfo(layer types.LayerType, gate
 		switch layer {
 		case types.Layer1:
 			messengerMessages[i].L1EventType = gatewayMessageMatch.L1EventType
```

Comment:
I think no need to change the event type, just `SentMessage` and `RelayedMessage` for messenger events.